Models/AzureFileProvider.cs

```diff
@@ -388,7 +388,7 @@ protected async Task<FileManagerResponse> RenameAsync(string path, string oldNam
                 {
                     FileManagerDirectoryContent directoryContent = fileItem;
                     isFile = directoryContent.IsFile;
-                    isAlreadyAvailable = isFile ? await IsFileExists(path + newName) : await IsFolderExists(path + newName);
```

Comment:
@Indhumathy-Loganathan  Ensure access control support

Reply:
I have ensured the access control support. The above fix doesn't affect the access control behavior.